streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -563,6 +545,17 @@ public void shouldSpecifyCorrectValueSerdeClassOnError() {
         }
     }
 
+    @Test
+    public void shouldThrowExceptionIfMaxInflightRequestsGreatherThanFiveIfEosEnabled() {
+        props.put(ProducerConfig.MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION, 7);
+        props.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, EXACTLY_ONCE);
+        final StreamsConfig streamsConfig = new StreamsConfig(props);
+        try {
+            final Map<String, Object> producerConfigs = streamsConfig.getProducerConfigs("clientId");
```

Comment:
nit: add a `fail(some message)` as if an exception is not thrown the test will "pass".

Reply:
Fixed